tools/jil/runner/index.js

```diff
@@ -134,6 +134,10 @@ function loadBrowsersAndRunTests () {
     } else {
       let sauceCreds = getSauceLabsCreds()
       connectionInfo = `http://${sauceCreds.username}:${sauceCreds.accessKey}@ondemand.saucelabs.com/wd/hub`
+
+      if (config.sauceExtendedDebugging && desired.browserName === 'chrome') {
```

Comment:
> Google Chrome (no older than 3 versions from latest) or Firefox browser (versions 53 and above) as the test browser

If the version is older than 3 from latest, there's no problem, right? What about including `firefox` too?

Reply:
>Extended Debugging generates additional assets that impact test performance and is not intended for everyday testing.

I didn't even see the line below including the extendedDebugging attribute. Thanks for pointing that out. Given the above note from the SL docs, I don't think we should have extended debugging on all the time. In fact, I wonder if having it turned off by default during CI will help speed the CI tests up any. Maybe just a small boost. I'll fix this in a bit.